rpm/python-ramalama.spec

```diff
@@ -1,51 +1,46 @@
-%global debug_package %{nil}
-
 %global pypi_name ramalama
-%global forgeurl https://github.com/containers/%{pypi_name}
+%global forgeurl  https://github.com/containers/%{pypi_name}
 # see ramalama/version.py
-%global version0 0.2.0
+%global version0  0.2.0
```

Comment:
@smooge do you need the version0 stuff at all? Packit will handle the version automatically in Fedora PRs. For regular builds, packit will use `git describe` to fetch the version by default.